src/Cohesity.Powershell/Scripts/ProtectionSource/Get-CohesityPhysicalAgentStatus.ps1

```diff
@@ -0,0 +1,35 @@
+function Get-CohesityPhysicalAgentStatus {
+    <#
+        .SYNOPSIS
+        Gets a list of the agent status of physical servers.
+        .DESCRIPTION
+        Gets a list of the agent status of physical servers.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Get-CohesityPhysicalAgentStatus
+        Returns the agent status of physcial servers.
+    #>
+    [OutputType('System.Array')]
+    [CmdletBinding()]
+    Param(
+    )
+
+    Begin {
+    }
+
+    Process {
+            $cohesityUrl = '/irisservices/api/v1/public/protectionSources/registrationInfo?includeEntityPermissionInfo=true&includeApplicationsTreeInfo=false&allUnderHierarchy=true&environments=kPhysical'
+            $resp = Invoke-RestApi -Method Get -Uri $cohesityUrl
+            if (200 -eq $Global:CohesityAPIStatus.StatusCode) {
+                $nodes = $resp.rootNodes
```

Comment:
Does these parse through nested objects or there are none when using the above filters ?

Reply:
The nodes are flat objects and no nested nodes.